src/getHumanAge.test.js

```diff
@@ -7,4 +7,29 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
```

Comment:
not bad, but try to add 3-4 more tests with your own ideas

Reply:
Done